app.py

```diff
@@ -279,6 +284,9 @@ def main(ARGS):
     parser.add_argument('-e', '--bg-video', metavar='VIDEO_FILE',
                         type=str, default=None,
                         help='Background video.')
+    parser.add_argument('-u', '--bg-url', metavar='VIDEO_FILE',
```

Comment:
Could you also change `metavar` to 'YOUTUBE_URL' or something like that?
```suggestion
    parser.add_argument('-u', '--bg-url', metavar='YOUTUBE_URL,
```